packages/generator/src/name-formatting-strategies.ts

```diff
@@ -51,15 +51,15 @@ function nextSuffix(name: string, previouslyGeneratedNames: string[]): number {
     previouslyGeneratedNames.filter(n => n.startsWith(name))
   );
   const lastElem = last(sortedList);
-  const match = lastElem ? lastElem.match(/(\d+)$/) : null;
+  const match = lastElem ? lastElem.match(/(?<!\d)(\d+)$/) : null;
```

Comment:
Just out of curiosity. The old regex said end with number the new said start with non numer (lookahead) and than end with number. Is it obvious why the evaluation is faster with the new one?

Reply:
question: I saw the reason why the old is slow from the link, but could you please answer my stupid question why the new one works?
My online friend (https://regexr.com) does not understand the new one;(